### ISSUE TITLE
Incompatibility with Catenary Replacer Updated

### ISSUE THREAD

**Issue by cylismaori** [boformer/NetworkSkins#56]
If you could modify: CatenaryUtils.cf from:
```
  private static readonly string[] DoubleCatenaryNames = new[]
        {
            "RailwayPowerline",
            "774449380.Catenary Type NL2A_Data",
            "774449380.Catenary Type NL2B_Data",
            "774449380.Catenary Type DE2A_Data",
            "774449380.Catenary Type PRR2A_Data",
            "774449380.Catenary Type PRR2B_Data",
            "774449380.Catenary Type JP2A_Data",
            "774449380.Catenary Type EXPO2A_Data",
        };

        private static readonly string[] SingleCatenaryNames = new[]
        {
            "RailwayPowerline Singular",
            "774449380.Catenary Type NL1A_Data",
            "774449380.Catenary Type NL1B_Data",
            "774449380.Catenary Type DE1A_Data",
            "774449380.Catenary Type PRR 1A_Data",
            "774449380.Catenary Type PRR 1A_Data",
            "774449380.Catenary Type JP1A_Data",
            "774449380.Catenary Type EXPO1A_Data",
        };
```
to:
```
   private static readonly string[] DoubleCatenaryNames = new[]
        {
            "RailwayPowerline",
            "774449380.Catenary Type NL2A_Data",
            "774449380.Catenary Type NL2B_Data",
            "774449380.Catenary Type DE2A_Data",
            "774449380.Catenary Type PRR2A_Data",
            "774449380.Catenary Type PRR2B_Data",
            "774449380.Catenary Type JP2A_Data",
            "774449380.Catenary Type EXPO2A_Data",
            "2444511901.Catenary Type NL2A_Data",
            "2444511901.Catenary Type NL2B_Data",
            "2444511901.Catenary Type DE2A_Data",
            "2444511901.Catenary Type PRR2A_Data",
            "2444511901.Catenary Type PRR2B_Data",
            "2444511901.Catenary Type JP2A_Data",
            "2444511901.Catenary Type EXPO2A_Data",
        };

        private static readonly string[] SingleCatenaryNames = new[]
        {
            "RailwayPowerline Singular",
            "774449380.Catenary Type NL1A_Data",
            "774449380.Catenary Type NL1B_Data",
            "774449380.Catenary Type DE1A_Data",
            "774449380.Catenary Type PRR 1A_Data",
            "774449380.Catenary Type PRR 1A_Data",
            "774449380.Catenary Type JP1A_Data",
            "774449380.Catenary Type EXPO1A_Data",
            "2444511901.Catenary Type NL1A_Data",
            "2444511901.Catenary Type NL1B_Data",
            "2444511901.Catenary Type DE1A_Data",
            "2444511901.Catenary Type PRR 1A_Data",
            "2444511901.Catenary Type PRR 1A_Data",
            "2444511901.Catenary Type JP1A_Data",
            "2444511901.Catenary Type EXPO1A_Data",
        };
```
This will restore compatibility.

**Comment by chameleon-tbn** [comment]
Added also

              // cylis's catenaries
            else if (prop.name.StartsWith("2444511901"))
            {
                //laneProp.m_position.y = -0.15f;
                laneProp.m_angle = 180f;

                if(Array.IndexOf(SingleCatenaryNames, prop.name) != -1)
                {
                    laneProp.m_position.x = 2f;
                }
            }

**Comment by boformer** [comment]
Fixed by f905b8bf0b2367691f993463d91b6027cbf4898c

Mod will be updated later this week
